metaG_metaT_integration/README.md

```diff
@@ -0,0 +1,11 @@
+# Integrating Different Types of Omics Data
+
+This folder includes two notebooks (in R and Python) that illustrate how metagenomic and metatranscriptomic data from the same sample can be retrieved from the [NMDC-runtime API](https://api.microbiomedata.org/docs) and integrated using KEGG annotations and taxonomy.
+
+## R 
+- [Static rendered Jupyter notebook](https://nbviewer.org/github/microbiomedata/nmdc_notebooks/blob/1316-create-multi-omics-notebook---metagenome-and-metatranscriptome/metaG_metaT_integration/R/metaG_metaT_omics_analysis.ipynb). This is the recommended way to interact with the notebook. _Viewing only, not editable_
```

Comment:
All links in this readme should point to the main branch.  That link will not work until it is merged in.  The links you have should be in the PR for review, but needs to be pointing to the main branch for merging (so it points to the main branch once it's merged).

Reply:
updated. Thank you.